apps/api/src/app.module.ts

```diff
@@ -45,7 +45,7 @@ const enterpriseImports = (): Array<Type | DynamicModule | Promise<DynamicModule
         modules.push(require('@novu/ee-auth')?.EEAuthModule);
       }
       if (require('@novu/ee-chimera')?.ChimeraModule) {
-        modules.push(require('@novu/ee-chimera')?.ChimeraModule);
+        // modules.push(require('@novu/ee-chimera')?.ChimeraModule);
```

Comment:
Reverting this temporary commented code.

```suggestion
        modules.push(require('@novu/ee-chimera')?.ChimeraModule);
```